src/wordpress-playground/worker-thread.ts

```diff
@@ -176,6 +176,11 @@ class WordPressPatcher {
 			`${DOCROOT}/wp-content/mu-plugins/add_requests_transport.php`,
```

Comment:
It was nice that `src/wordpress-playground/worker-thread.ts` live reloaded and was easy to work with.

Which files are live reloaded, and which require a full Docker rebuild?

Reply:
@danielbachhuber anything you do via a `.ts` file is live-reloaded. Anything you do via a Dockerfile requires a rebuild.